justfile

```diff
@@ -0,0 +1,35 @@
+default:
+  @just --list
+
+create-cluster:
+  kind create cluster --config ./kubernetes/kind-cluster-config.yml
+
+deploy-ingress-controller:
+  kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+
+wait-for-ingress-controller:
+  kubectl wait --namespace ingress-nginx --for=condition=ready pod --selector=app.kubernetes.io/component=controller --timeout=600s
+
+deploy-astria:
+  kubectl apply -k kubernetes/
+
+cleanup-astria: delete-cluster delete-ingress delete-astria
```

Comment:
The order in this does not make sense: if you delete the cluster none of the other commands should even work.